dart/lib/src/enricher/io_enricher_event_processor.dart

```diff
@@ -0,0 +1,106 @@
+import 'dart:async';
+import 'dart:io';
+
+import '../protocol.dart';
+import '../sentry_options.dart';
+
+EventProcessor enricherEventProcessor(SentryOptions options) {
+  final processor = IoEnricherEventProcessor(
+    options.platformChecker.hasNativeIntegration,
+    options.sendDefaultPii,
+  );
+  return processor.apply;
+}
+
+/// Enriches [SentryEvents] with various kinds of information.
+/// Uses Darts [Platform](https://api.dart.dev/stable/dart-io/Platform-class.html)
+/// class to read information.
+class IoEnricherEventProcessor {
```

Comment:
All of these enrichers are kinda `EventProcessor` classes. I created https://github.com/getsentry/sentry-dart/issues/479 . So this should be pretty straight forward to migrate in the future.

Reply:
sounds good, so we can get unblocked with this, I will do a fine review tomorrow